src/Microsoft.DotNet.Cli.Utils/Muxer.cs

```diff
@@ -11,7 +11,7 @@ public class Muxer
 
         private string _muxerPath;
 
-        internal string SharedFxVersion
+        public string SharedFxVersion
```

Comment:
Why turn this public. This is not meant to be consumed outside of the CLI repo, which is why it is internal.